src/react/components/common/keyboardManager/keyboardRegistrationManager.ts

```diff
@@ -21,40 +22,31 @@ export type KeyboardEventHandler = (evt?: KeyboardEvent) => void;
 export class KeyboardRegistrationManager {
     private registrations: IKeyboardRegistrations = {};
 
-    /**
-     * Registers a keyboard event handler for the specified key code
-     * @param keyEventType Type of key event (keydown, keyup, keypress)
-     * @param keyCodes a list of key code and key code combinations, ex) Ctrl+1
-     * @param handler The keyboard event handler
-     *
-     * @returns a function for deregistering the handler
-     */
-    public addHandler(keyEventType: KeyEventType, keyCodes: string[], handler: KeyboardEventHandler): () => void {
-        Guard.null(keyEventType);
-        Guard.expression(keyCodes, (keyCodes) => keyCodes.length > 0);
-        Guard.null(handler);
+    public registerBinding = (binding: IKeyboardBindingProps) => {
+        Guard.null(binding.keyEventType);
+        Guard.expression(binding.accelerators, (keyCodes) => keyCodes.length > 0);
+        Guard.null(binding.handler);
```

Comment:
Because the method you deleted had JSDocs, it may be helpful for this one to have JSDocs too